trackio/__init__.py

```diff
@@ -65,6 +67,8 @@ def init(
             case the Space will be created in the currently-logged-in Hugging Face
             user's namespace. If the Space does not exist, it will be created. If the
             Space already exists, the project will be logged to it.
+        space_storage ([`~huggingface_hub.SpaceStorage`] or `None`, *optional*, defaults to `None`):
+            Choice of persistent storage tier.
```

Comment:
Perhaps its worth listing the options from the enum explicitly here in the docstring